TheModernIlluminati/TheModernIlluminati/Pages/Laureate.cshtml

```diff
@@ -158,48 +158,70 @@
 
     <div>
         <h1 class="display-4">Laureates</h1>
+        <style>
```

Comment:
Added styling and right spacing between columns to make the table content appear neatly.
https://www.w3schools.com/css/

Reply:
Good idea.  Go one step further with this, and put it in a linked stylesheet.